build.sh

```diff
@@ -58,6 +58,33 @@ usage()
   echo "Arguments can also be passed in with a single hyphen."
 }
 
+function SetBranding()
+{
+  local brandingValue="$1"
+  
+  if [[ "$brandingValue" == "preview" ]]; then
```

Comment:
I know this was existing code.... Where to RCs fit?  They are in a way a middle ground between preview and rtm especially with a golive license.

Reply:
They're still "preview".